AudibleBoulders/AudibleBoulders
# Updated dashboard page to update user status view

#### What's this PR do?
1) Updates the user view in dashboard page to show current user status. 
2) Added bootstrap and glyphicons. 
3) Modified db query in dashboards-org-repo js file to include set_up and up_to_date fields in query. 
4) Added additional css to style user view.

#### What are the important parts of the code?
N/A

#### How should this be tested by the reviewer?
1) Pull down and test by navigating to the dashboard page. 
2) Make sure to have at least 3 different dashboards (use 3 different repos). 
3) Update the set_up and up_to_date fields in user_dashboards table by using the following query; modify values as needed: 
###### *UPDATE users_dashboards SET set_up='1', up_to_date='1' WHERE id='3';* 

4)You can use the following sql query to add in a test diff; modify values as needed: 
###### *INSERT INTO diffs (users_dashboards_id, file, mod_type, commit_message) VALUES ('3', 'index.js', 'A', 'that new new');*

#### Is any other information necessary to understand this?
N/A

#### What are the relevant tickets? (Please add `closes`, `refs`, etc)
N/A

#### Screenshots (if appropriate)
